scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -187,6 +187,10 @@ public boolean isSubmitPossible() {
         return status.isSubmittable();
     }
 
+    public boolean isPaused() {
+        return status == SchedulerStatus.PAUSED;
```

Comment:
should apply also to SchedulerStatus.STOPPED (change the method name accordingly)